modules/vocabularies/src/main/java/ca/sickkids/ccm/lfs/vocabularies/internal/BioOntologyRepositoryHandler.java

```diff
@@ -58,6 +60,16 @@
     private static final String REQUEST_CONFIGURATION =
         "?display_context=false&display_links=false&apikey=";
 
+    /** The list of vocabulary format names that are not supported by the current parser. Such
+        vocabularies should be explicitly requested to be downloaded in some other supported format. */
+    private static final Set<String> UNSUPPORTED_VOCABULARY_FORMATS =
```

Comment:
This is not correct, since OWL is a supported, what is not supported is some of its representation formats. Unfortunately, there is no way to detect the default download format, so we request RDF whenever we see that an ontology comes from an OWL file.

There are OBO vocabularies that also have a RDF version for download, and there are, unfortunately, some OWL vocabularies that don't have a RDF version.
I don't have a perfect name for this, so how about:

```suggestion
    private static final Set<String> RDF_VOCABULARY_FORMATS =
```
Or:

```suggestion
    private static final Set<String> VOCABULARY_FORMAT_SUPPORTING_RDF =
```